docs/Frontend_tutorials/FE-Testing.md

```diff
@@ -35,24 +35,36 @@ npm test -- query.spec.js
 
 ---
 
-## Cypress tests
+## Playwright tests
 
-We are using Cypress for testing our Frontend.
+We are using Playwright for testing our Frontend.
 In this section we will try to answer few questions, for example:
-*How to write CY tests? How we manage them and how to run them?*
+*How to write Playwright tests? How we manage them and how to run them?*
 
-### Getting started with Cypress
-At the beginning please read some basics about Cypress:
+### Getting started with Playwright
+At the beginning please read some basics about Playwright:
 
-1. [Installing Cypress](https://docs.cypress.io/guides/getting-started/installing-cypress)
-2. [Assertions](https://docs.cypress.io/guides/references/assertions)
-3. And most importantly - [Best practices](https://docs.cypress.io/guides/references/best-practices).
+1. [Installing Playwright](https://playwright.dev/docs/intro#installing-playwright)
+2. [Assertions](https://playwright.dev/docs/test-assertions)
+3. And most importantly - [Best practices](https://playwright.dev/docs/selectors#best-practices).
 
 ### Structure
 
-On the platform, Cypress tests are located in `/cypress/integration` folder. The main configuration of cypress is in `/cypress.json file`.
+On the platform, Playwright tests are located in `/playwright` folder. The main configuration of Playwright is in `/playwright.config.ts` file.
 
-Cypress itself runs JS tests in its own syntax against some running server - this is specified in `/cypress.json`.
+Playwright itself runs JS tests in its own syntax against some running server - this is specified in `/playwright.config.ts`.
+
+1. ./playwright/fixtures
+
+- This folder contains JSON files representing entities of our system. Both existing once and the new ones.
```

Comment:
```suggestion
- This folder contains JSON files representing entities in our system. If a JSON contains an ID the entity exists in the testing database, otherwise they're mostly used for filling inputs.
```